File: ungs-docsys-ms/src/main/java/com/ungs/docsys/dtos/UserInfoResponseDto.java
package com.ungs.docsys.dtos;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDate;

@Data
@Builder
public class UserInfoResponseDto {
    private String email;
    private String firstName;
    private String lastName;
    private String identificationType;
    private String identificationNumber;
    private String cuilCuit;
    private String phone;
    private LocalDate birthDate;
    private String nationality;
}

File: ungs-docsys-bff/src/modules/login/user-info/dtos/user-info-response.dto.ts
import { ApiProperty } from "@nestjs/swagger";

export class UserInfoResponseDto {
    @ApiProperty()
    email: string;
    @ApiProperty()
    firstName: string;
    @ApiProperty()
    lastName: string;
    @ApiProperty()
    identificationType: string;
    @ApiProperty()
    identificationNumber: string;
    @ApiProperty()
    phone: string;
    @ApiProperty()
    birthDate: Date;
    @ApiProperty()
    nationality: string;
    @ApiProperty()
    cuilCuit: string;
}
